src/LibraryUtilities.ts

```diff
@@ -38,6 +38,11 @@ export interface IncludeInfo {
     inclusive?: boolean;
 }
 
+export class IncludeItemPair {
+    include: IncludeInfo;
+    parentItem: ItemData;
+}
```

Comment:
The term *parent* gives a confusing meaning in hierarchical relationship, I didn't understand this until I read the code bellow, and now I think a better name would be `owningItem`. Does that make sense?